### ISSUE TITLE
Incompatibility with neo4j-driver==1.6.0

### ISSUE THREAD

**Issue by stevenvdr** [neo4j-contrib/neomodel#344]
`neo4j.v1.types.graph.Node` changed it's internal property from `self.properties` to `self._properties`, causing issues with compatibility.

What is the roadmap for neo4j-driver support? 

Fixed locally by:
```
class FixedStructuredNode(StructuredNode):

    @classmethod
    def inflate(cls, node):
        node.properties = node._properties
        return super().inflate(node)
```

However there could be other incompatibilities.

**Comment by robinedwards** [comment]
Hey @stevenvdr if you raise a PR with the change our testsuite should hopefully catch most of the potential issues

**Comment by stevenvdr** [comment]
Alright, I'll see if I can free some time to do that. 

How about older versions of neo4j-driver? Do we still want to support those? Or would 1.6.0 become the only versions that's supported?

**Comment by stevenvdr** [comment]
Fixed by #347. 

Thanks @mprahl ! You beat me to it :)

**Comment by mprahl** [comment]
@robinedwards could you please release a new version of neomodel with #347?
